Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -283,24 +283,25 @@ function vtkWidgetManager(publicAPI, model) {
     // Default cursor behavior
     model._apiSpecificRenderWindow.setCursor(widget ? 'pointer' : 'default');
 
+    let wantRender = false;
     if (model.widgetInFocus === widget && widget.hasFocus()) {
       activateHandle(widget);
-      // Ken FIXME
-      model._interactor.render();
-      model._interactor.render();
+      wantRender = true;
     } else {
       for (let i = 0; i < model.widgets.length; i++) {
         const w = model.widgets[i];
         if (w === widget && w.getNestedPickable()) {
           activateHandle(w);
           model.activeWidget = w;
+          wantRender = true;
         } else {
+          wantRender ||= !!w.get().activeState;
```

Comment:
If the widget had an active state prior to deactivating, we request a render. `vtkAbstractWidget` doesn't have a `getActiveState()` API; is this something worth adding?

Reply:
probably making that call in get() is not great...

Reply:
I think we should avoid using `get()`

Reply:
Agreed. I think exposing `activeState` is preferred. It does mean that if by some chance someone decides to activate a widget substate manually (i.e. not through widget.activateHandle), then it won't be rendered.